webapp: Appservice plan update supports Isolated SKU

panchagnula opened the pull request:
Fixes #5760
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

panchagnula commented:
@qwordy can i get a PR on this please? Thank you.

panchagnula commented:
@qwordy can I get a PR please - this already missed the original release milestone because of the delay in PR. Thank you.

qwordy commented on src/azure-cli/HISTORY.rst:
```diff
@@ -55,7 +61,6 @@ Release History
 
 **AppService**
 
-* Added "webapp config access-restriction show | set | add | remove"
```
I'm not sure whether we should modify history of old versions.

panchagnula commented:
The reason i added no test cases is because this SKU supported on ASE resource types only & this is not something we currently support on CLI. We just added the option to scale an ASP using CLI for now.
